conf/messages

```diff
@@ -493,6 +493,60 @@ otherBeneficiary.whenRemoved.error.invalid = Enter a real date when the benefici
 otherBeneficiary.whenRemoved.error.future = The date the beneficiary was removed from the trust must be today or in the past
 otherBeneficiary.whenRemoved.error.past = The date the beneficiary was removed must be on or after the date they were added
 
+otherBeneficiary.description.title = What is the description of the other type of beneficiary?
+otherBeneficiary.description.heading = What is the description of the other type of beneficiary?
+otherBeneficiary.description.checkYourAnswersLabel = What is the description of the other type of beneficiary?
+otherBeneficiary.description.hint = This must be 56 characters or less. For example, future grandchildren and future great-grandchildren.
+otherBeneficiary.description.error.required = Enter the description the other type of beneficiary
```

Comment:
Enter the description of the beneficiary

Reply:
Should now be "other beneficiary". Done.